recipes/ngscheckmate/meta.yaml

```diff
@@ -1,33 +1,40 @@
 package:
   name: ngscheckmate
-  version: 1.0.0
+  version: 1.0.1
 
 source:
-  url: https://github.com/parklab/NGSCheckMate/archive/8ea2c043896adf9bbd1dc5bb2136f3a1dab9b445.zip
-  md5: 4c4e34abc8c4851d29661f5f0cc13b49
+  url: https://github.com/parklab/NGSCheckMate/archive/51e2634ec50d659980dcb2f3274046d4219fabc7.zip 
+  md5: a2caabb0fa5f4db43af8700b17ff1b7f
 
 build:
-  number: 4
+  number: 0
   noarch: generic
 
 requirements:
   host:
-    - python 2.7.18
-    - R 4.1
-    - samtools 1.14
+    - python 2.*
+    - R 4.0
```

Comment:
this should be r-base ... is the pin to version 4.0 really needed?

Reply:
I've now changed this to r-base and relaxed the version requirements